spec/fixtures/project-content/index.js

```diff
@@ -1,3 +1,5 @@
 import text from './text.txt'
 
-console.log(text)
+const log = () => console.log(text)
+
+log()
```

Comment:
I wanted to "make sure" it was being transpiled by Babel. I'll improve it before merging.